galley/pkg/config/processor/transforms/ingress/dataset_test.go

```diff
@@ -166,6 +169,8 @@ func vs1() *resource.Entry {
 				},
 			},
 		},
+		// Rationale: Gomega will insist on typed nil, but output only 'nil' on failure.
+		Origin: (*rt.Origin)(nil),
```

Comment:
@sushicw I left a comment the first time, but not everywhere I used the technique.  Putting it everywhere seemed like too much.  Thoughts?